tests/e2e/policy_driven_vol_allocation.go

```diff
@@ -395,9 +398,353 @@ var _ = ginkgo.Describe("[vol-allocation] Policy driven volume space allocation
+		framework.Logf("File system size after expansion : %v", fsSize)
+		// Filesystem size may be smaller than the size of the block volume
+		// so here we are checking if the new filesystem size is greater than
+		// the original volume size as the filesystem is formatted for the
+		// first time
+		gomega.Expect(fsSize > originalFsSize).To(gomega.BeTrue(),
+			fmt.Sprintf(
+				"filesystem size %v is not > than before expansion %v for pvc %q",
+				fsSize, pvclaim.Name, originalFsSize))
+
+		framework.Logf("File system resize finished successfully")
+		ginkgo.By("Verify that the expanded CNS volume is compliant")
+		storagePolicyMatches, err = e2eVSphere.VerifySpbmPolicyOfVolume(volumeID, policyName)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Expect(storagePolicyMatches).To(gomega.BeTrue(), "storage policy verification failed")
+		e2eVSphere.verifyVolumeCompliance(volumeID, true)
+
+	})
+
+	/*
+		Verify online LZT/EZT volume expansion of attached volumes with IO
+		Steps:
+		1	Create a SPBM policies with EZT and LZT volume allocation
+		2	Create SCs using policies created in step 1 and allowVolumeExpansion set to true
+		3	Create 2g PVCs using each of the SCs created in step 2
+		4	Wait and verify for pvcs to be bound
+		5	Create a pod using pvcs say pod1 and wait for it to be ready
+		6	Expand pvcs while writing some data on them
+		7	Wait and verify the file system resize on pvcs
+		8	Verify the data on the PVCs match what was written in step 7
+		9	Delete pod1
+		10	Delete pvc1
+		11	Delete the SC created in step 2
+		12	Deleted the SPBM policy created in step 1
+	*/
+	ginkgo.It("[csi-block-vanilla][csi-block-vanilla-parallelized] Verify online LZT/EZT volume expansion of "+
+		"attached volumes with IO", func() {
+
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		sharedvmfsURL := os.Getenv(envSharedVMFSDatastoreURL)
+		if sharedvmfsURL == "" {
+			ginkgo.Skip(fmt.Sprintf("Env %v is missing", envSharedVMFSDatastoreURL))
+		}
+
+		scParameters := make(map[string]string)
+		policyNames := []string{}
+		pvcs := []*v1.PersistentVolumeClaim{}
+		pvclaims2d := [][]*v1.PersistentVolumeClaim{}
+		scs := []*storagev1.StorageClass{}
+
+		rand.Seed(time.Now().UnixNano())
+		suffix := fmt.Sprintf("-%v-%v", time.Now().UnixNano(), rand.Intn(10000))
+		categoryName := "category" + suffix
+		tagName := "tag" + suffix
+
+		catID, tagID := createCategoryNTag(ctx, categoryName, tagName)
+		defer func() {
+			deleteCategoryNTag(ctx, catID, tagID)
+		}()
+
+		attachTagToDS(ctx, tagID, sharedvmfsURL)
+		defer func() {
+			detachTagFromDS(ctx, tagID, sharedvmfsURL)
+		}()
+
+		allocationTypes := []string{
+			eztAllocType,
+			lztAllocType,
+		}
+
+		ginkgo.By("create SPBM policies with LZT, EZT volume allocation respectively")
+		ginkgo.By("Create SCs using policies created in step 1")
+		ginkgo.By("create a large PVC each using the storage policies created from step 2")
```

Comment:
Please correct this statement as we are not creating pvc of large size here

Reply:
done